libmscore/beam.cpp

```diff
@@ -283,6 +283,20 @@ bool Beam::twoBeamedNotes()
       return true;
       }
 
+//---------------------------------------------------------
+//   totalTicks
+//    calculate the ticks of all chords and rests connected by the beam
+//---------------------------------------------------------
+
+Fraction Beam::totalTicks() const
+      {
+	        Fraction totalTicks = Fraction(0, 1);
```

Comment:
Indentation issues...

Reply:
Other than that: yes, refactoring this way is good

Reply:
could even be a static function, as it isn't used/needed anywhere else?

Reply:
Oh... Seems like VS has a different indent principle, I have to manually enter spaces as the automatic indent is made up of tabs...

Reply:
I actually forget how to make a function static... haven't used that in a very long time.

Reply:
prefix it with the keyword `static` (and remove it from the .h file)

Reply:
Not sure though we want this here, @anatoly-os ?

Reply:
remove the whole function statement from .h file??

Reply:
Yes, for a static function,  which is not visible to others including that header this is the way to go. Also in the cpp file it needs to be defined before use, bit that is the case here already

Reply:
OK, but why bother? Maybe this function can be used later... when @mattmcclinch fixed #284481 he probably didn't know I was going to use his isOnlyDeletedRests() in this fix :D

Reply:
True, that's why I'm asking @anatoly-os's opinion ;-)